.gitignore

```diff
@@ -1 +1,8 @@
-.DS_Store
\ No newline at end of file
+.DS_Store
+.idea
```

Comment:
Let's remove `.idea` - IDE is a user choice so should be ignored with global gitignore